commit c443fc3b646bbcb0c7b38fe08ab177b8be3ebb0c (refs/changes/07/4807/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-09-07T10:50:37-07:00 (1 year, 1 month ago)
    
    TRITON-760 imgapi-external-manta-setup and imgapi-manta-setup were broken by sdc-clients 11.x update in TRITON-53

diff --git a/CHANGES.md b/CHANGES.md
index e4bba77..b3c9129 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,10 @@
 # IMGAPI changelog
 
+## 4.5.1
+
+- TRITON-760 Fix imgapi-external-manta-setup and imgapi-manta-setup tooling
+  broken in v4.4.0 by TRITON-53.
+
 ## 4.5.0
 
 - TRITON-52 x-DC image copy. This adds a new ImportImageFromDatacenter
diff --git a/bin/imgapi-external-manta-setup b/bin/imgapi-external-manta-setup
index 1e6bd42..d9b5a31 100755
--- a/bin/imgapi-external-manta-setup
+++ b/bin/imgapi-external-manta-setup
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -142,6 +142,7 @@ function main() {
 
     var sapi = new sdcClients.SAPI({
         url: 'http://' + config.sapi_service,
+        version: '~2',
         log: log,
         agent: false
     });
diff --git a/bin/imgapi-manta-setup b/bin/imgapi-manta-setup
index e7843fd..c89f8db 100755
--- a/bin/imgapi-manta-setup
+++ b/bin/imgapi-manta-setup
@@ -6,16 +6,16 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
  * Setup this IMGAPI to use a Manta (for storage) that shares the same master
- * UFDS as this IMGAPI. This IMGAPI+Manta setup (provide the Manta is local,
- * i.e. not across the WAN) is the preferred setup.: local Manta, uses the
- * $ufds_admin_login (typically the "admin" login) user.
+ * UFDS as this IMGAPI. This IMGAPI+Manta setup (provided the Manta is local,
+ * i.e. not across the WAN) is the preferred setup. It uses the
+ * $ufds_admin_login (typically the "admin" login) account.
  *
- * The *alternative* (e.g. to have IMAGE use a Manta with a separate UFDS
+ * The *alternative* (e.g. to have IMGAPI use a Manta with a separate UFDS
  * database) is to use the 'imgapi-external-manta-setup' script in this same
  * directory.
  *
@@ -62,7 +62,7 @@ var bunyan = require('bunyan');
 var log; // set in `main()`
 
 var USAGE = (
-    'Setup IMGAPI in this SmartDataCenter to use a Manta sharing the same\n'
+    'Setup IMGAPI in this TritonDC to use a Manta sharing the same\n'
     + 'master UFDS. Preferrably the Manta is local (i.e. not across the WAN).\n'
     + 'This will do the appropriate setup for all instances of IMGAPI in this\n'
     + 'DC. This is also re-runnable if it fails part way.\n'
@@ -134,6 +134,7 @@ function getMantaUrl(mantaUrl, sdcApp, callback) {
 
     var sapi = new sdcClients.SAPI({
         url: 'http://' + sdcApp.metadata.SAPI_SERVICE,
+        version: '~2',
         log: log,
         agent: false
     });
diff --git a/package.json b/package.json
index df812be..69dd597 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "4.5.0",
+  "version": "4.5.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
